### DISCUSSION TITLE
Subdomain from Hostinger to a Vercel Site

### DISCUSSION THREAD

**Discussion by KoesterJannik** [vercel/community#805] [Help] [answered]
Hey, i have a domain bought on hostinger and so it has their nameservers.

Is it possible to set the nameservers for a subdomain to the vercel nameservers?

**Comment by amyegan** [comment, marked as answer]
Hi @KoesterJannik! It's possible through NS records, but chaining DNS lookups introduces a potential point of failure in name resolution, makes debugging more difficult, and can make the name resolution process take longer. If you really need to do it that way, you could use something like this: https://webmasters.stackexchange.com/a/125997.

Instead, I would recommend that you [point the subdomain at your Vercel Project using a DNS record](https://vercel.com/docs/concepts/projects/custom-domains#step-4:-configuring-the-domain). 

Some pages that might be helpful:
- [Custom Domains](https://vercel.com/docs/concepts/projects/custom-domains#)
- [How do I add a custom domain to my Vercel project?](https://vercel.com/guides/how-do-i-add-a-custom-domain-to-my-vercel-project)
- [How can I debug a DNS record?](https://vercel.com/guides/how-can-i-debug-a-dns-record#apex-domain-and-a-records)
- [How to manage my DNS records at Hostinger?](https://support.hostinger.com/en/articles/1583249-how-to-manage-my-dns-records-at-hostinger)
